src/server.ts

```diff
@@ -181,10 +184,24 @@ const fetchVulnerabilities = async (reqData) => {
     const headers = {
         'Content-Type': 'application/json',
         'Authorization': 'Bearer ' + config.api_token,
+        'request_id': uuid.v4(),
     };
+
+    let manifestHash = crypto.createHash("sha1").update(reqData.filepath).digest("hex");
+    url += `&utm_content=${manifestHash}`;
+
+    if (config.source) {
+        url += `&utm_source=${config.source}`;
+    }
+
     if (config.uuid) {
         headers['uuid'] = config.uuid;
     }
+
+    if (config.user_agent) {
+        headers['User-Agent'] = config.user_agent;
+    }

```

Comment:
As mentioned, don't touch User-Agent, utm_content and utm_source is enough

Reply:
Done.